tests/plugins/pastefromword/manual/copypasteimage.md

```diff
@@ -0,0 +1,12 @@
+@bender-tags: tc, 4.6.0
```

Comment:
Please add ticket number `16912` as a tag too. Also correct target version to 4.7.0